promql/testdata/functions.test

```diff
@@ -19,3 +19,11 @@ eval instant at 50m resets(http_requests[50m])
 	{path="/bar"} 1
 
 eval instant at 50m resets(nonexistent_metric[50m])
+
+clear
+
+load 5m
+	http_requests	0+10x10
```

Comment:
Would be nice to do a reset somewhere in here, to test if it's handled correctly.